package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "hof-component-date",
+  "version": "1.0.0",
+  "description": "A hof middleware for rendering and processing 3 part date fields",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/UKHomeOfficeForms/hof-component-date"
+  },
+  "scripts": {
+    "lint": "eslint .",
+    "test": "mocha"
+  },
+  "author": "UKHomeOffice",
+  "license": "MIT",
+  "dependencies": {
+    "async": "^2.1.4",
+    "hof-controllers": "^5.0.0",
+    "hof-form-controller": "^2.0.0",
+    "hof-form-wizard": "^1.1.0",
```

Comment:
I don't think any of the `hof-*` dependencies are needed, and given how this should fit into a dependency tree, probably *shouldn't* be needed.